features/support/github.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 
 const Github = require('github');
-const github = new Github();
+const github = new Github({
+    host: process.env.TEST_SCM_HOSTNAME || 'api.github.com',
+    pathPrefix: process.env.TEST_SCM_HOSTNAME ? '/api/v3' : ''
```

Comment:
should probably set a default value for `protocols` as well?

Reply:
added!